### PR TITLE
Migrate to Axios

### PR DESCRIPTION
**Pull request by LissetteIbnz**
#169 Migrate sample 12 Table Table with Axios

### CODE REVIEW

File: 12_TableHttp_Axios/readme.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+      };
+
+      return member;
+    });
+
+    return members;
+  }
+}
+
+export const memberAPI = new MemberAPI();
```
Comment (brauliodiez) [inline 12_TableHttp_Axios/readme.md]: I thini its a good time to remove the class/singleton approach and use just a pure functional approach, what"s your take on this?

Comment (brauliodiez) [reply 12_TableHttp_Axios/readme.md]: A sample of this functional approach: https://github.com/Lemoncode/componentization-examples/blob/master/00_Hotels_List/01_implementation/src/api/hotel.ts

When we implemented the class/singleton approach was a mistake from our side

Comment (LissetteIbnz) [reply 12_TableHttp_Axios/readme.md]: Great! I agree with you. In my job I do it.
Perhaps should I add Axios as vendor in webpack config? what's your opinion about this?

Comment (brauliodiez) [reply 12_TableHttp_Axios/readme.md]: Good point, on thw other hand you can remove the return and the let variable

Comment (LissetteIbnz) [reply 12_TableHttp_Axios/readme.md]: Perfect. I hope finish in a minutes...

**Comment by brauliodiez** [comment]
Good stuff, thanks!

**Pull request merged**